lib/shrine/plugins/zencoder.rb

```diff
@@ -17,7 +17,7 @@ def perform data, payload
 
       module AttacherMethods
         def start_encoding data
-          ::Zencoder::Job.create input: url,
+          ::Zencoder::Job.create input: url(:original),
```

Comment:
@samharnack: What I found out from testing it supports cached version - it simply returns cached url.
I think we can use it as default, as we use 'original' as asset version in the above plugin.